### PR TITLE
Attempt to fix circular references.

### PR DESCRIPTION
**Pull request by haringsrob**
Based on the issues in https://github.com/phpactor/phpactor/issues/1791, this pr attemts to resolve it.

I have little to no knowledge about the internals nor have I tested this with tests.

It seems to work but might not at all.

### CODE REVIEW

**Comment by haringsrob** [comment]
Screenshot 2022-07-18 at 14 51 02

Seems to work.

File: lib/WorseReflection/Bridge/Phpactor/MemberProvider/MixinMemberProvider.php [hunk 1]
```diff
@@ -30,12 +32,25 @@ public function provideMembers(ServiceLocator $locator, ReflectionClassLike $cla
                 continue;
             }
 
+            $visited = $reflection->getClass()->getVisited();
```
[the remark this answers is not in the document]
Comment (haringsrob) [reply lib/WorseReflection/Bridge/Phpactor/MemberProvider/MixinMemberProvider.php:35]: As I mentioned, I do not really know the internals that well. Just tried to fix this one issue.

Perhaps we can add $reflection->getClass()->wasVisited($SomeClass) instead?

**Comment by dantleech** [comment]
will be fixed properly by #1821

**Comment by vlada-dudr** [comment]
Sorry for delay and thank you for guidance.

**Pull request closed without merging**